docs/contributing/coding-standards/component-options.md

```diff
@@ -4,204 +4,42 @@ Configuration options allow developers to pass additional information into a com
-  }
-}
-```
-
-Within the entry class, you'll then create an object with the default configuration for the component. These are the options the component will use if no other options are provided.
-
-```mjs
-export class Accordion {
-  static defaults = Object.freeze({
-    rememberExpanded: true
-  })
-}
-```
-
-Next, use the `mergeConfigs` helper to combine the default config and the configuration object. The result of the merge is assigned globally (using `this`) so that it's accessible anywhere within the component's JavaScript.
-
-The order in which variables are written defines their priority, with objects passed sooner being overwritten by those passed later. As we want the user's configuration to take precedence over our defaults, we list our default configuration object first.
-
-There is no guarantee `config` will have any value at all, so we set the default to an empty object (`{}`) in the constructor parameters.
-
-```mjs
-import { mergeConfigs } from '../../common/configuration.mjs'
-
-export class Accordion {
-  constructor($root, config = {}) {
-    this.config = mergeConfigs(
-      Accordion.defaults,
-      config
-    )
-  }
-}
-```
-
-We can now reference the configuration option anywhere we need to in the component's JavaScript:
-
-```js
-this.config.rememberExpanded
-// => true
-```
-
-It's now possible to individually initialise the component with configuration options.
-
-```html
-<script type="module">
-  import { Accordion } from '{path-to-javascript}/govuk-frontend.min.js'
-
-  const $element = document.getElementById('accordion')
-
-  new Accordion($element, {
-    rememberExpanded: false
-  })
-</script>
-```
-
-## Allowing options to be passed through the `initAll` function
-
-Often, teams will not be individually initialising components. Instead, GOV.UK Frontend ships with an `initAll` function, which searches the page for instances of components and automatically initialises them.
-
-In `packages/govuk-frontend/src/govuk/init.mjs`, update the component's `new` class to pass through a nested configuration object. The nested object should use the component's name converted to camelCase (for example, the 'Character Count' component becomes `characterCount`).
-
-```js
-new Accordion($accordion, config.accordion)
-```
-
-It's now possible to pass configuration options for your component, as well as multiple other components, using the `initAll` function.
-
-```html
-<script type="module">
-  import { initAll } from '{path-to-javascript}/govuk-frontend.min.js'
-
-  initAll({
-    accordion: {
-      rememberExpanded: false
-    },
-    characterCount: {
-      threshold: 90
-    }
-  })
-</script>
-```
-
-## Adding support for HTML `data-*` attributes
-
-For convenience, we also allow for configuration options to be passed through HTML `data-*` attributes.
+For more information, see our [guidance on building configurable components](https://frontend.design-system.service.gov.uk/building-configurable-components/).
```

Comment:
**issue** The new documentation is a section of the existing 'Building your own JavaScript components' page.

```suggestion
For more information, see our [guidance on building configurable components](https://frontend.design-system.service.gov.uk/building-your-own-javascript-components/#building-configurable-components).
```